src/mlpack/methods/det/dt_utils_impl.hpp

```diff
@@ -183,10 +183,10 @@ DTree<MatType, TagType>* Trainer(MatType& dataset,
   timers.Start("cross_validation");
   // Go through each fold.  On the Visual Studio compiler, we have to use
   // intmax_t because size_t is not yet supported by their OpenMP
-  // implementation. omp_size_t is the appropriate type according to the
+  // implementation. size_t is the appropriate type according to the
   // platform.
```

Comment:
As `omp_size_t` has been changed to `size_t`, suggest to remove all text after "Go through each fold."  
Alternatively, simply remove the entire comment block, as the loop definition clearly indicates that we're going through each fold.

Reply:
Agreed, the comment is now outdated.  I removed it entirely in b6b6123910.